media-libs/libgroove/libgroove-4.3.0.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit cmake-utils
+
+DESCRIPTION="Streaming audio processing library."
+HOMEPAGE="https://github.com/andrewrk/libgroove"
+SRC_URI="https://github.com/andrewrk/libgroove/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+chromaprint +loudness +sound"
+
+DEPEND="media-video/ffmpeg
```

Comment:
* Upstream has some suggested flags, but from my tests they are all optional runtime dependencies.
* Support for libav was recently removed, but it looks like this version still supports it. I think the default should be ffmpeg, to minimize problems when upgrading to the next version, unless @andrewrk thinks libav is better for 4.3.0.